config/capz/manager_pull_policy.yaml

```diff
@@ -8,4 +8,4 @@ spec:
     spec:
       containers:
       - name: manager
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
```

Comment:
Same as above. I think these get changed when you run `make tilt-up`

Reply:
yes, I think so... I'll remove it and squash commits too